src/testdrive/src/action/kafka/verify.rs

```diff
@@ -28,14 +28,14 @@ pub enum SinkFormat {
     Json { key: bool },
 }
 
-pub enum SinkConsistencyFormat {
-    Debezium,
+pub enum VerifySource {
```

Comment:
nit: this reads like "verify source!" but is intended as "verification source". At least to me, what I'm also not a native speaker, so ... :man_shrugging:

Reply:
Disregard this, I now saw the later commit. :ok_hand: